framework/src/main/java/org/apache/drill/test/framework/Utils.java

```diff
@@ -533,72 +577,68 @@ public static String getSqlResult(ResultSet resultSet) throws SQLException {
     }
     return nullabilitiesInStrings;
   }
-  
+
   /**
-   * Saves content of existing drill storage plugins.
-   * 
-   * @param ipAddress
-   *          IP address of node to update storage plugin for
-   * @param pluginType
-   *          type of plugin; e.g.: "dfs", "cp"
-   * @return content of the specified plugin
-   * @throws Exception
+   * Update storage plugins in directory
+   * @param templatePath directory having drill storage plugin templates
+   * @throws InterruptedException
    */
-  public static String getExistingDrillStoragePlugin(String ipAddress,
```

Comment:
Have we removed this method?

Reply:
yeah we don't use it anywhere. i believe we wouldn't need it for our testing

Reply:
No, I think we do need it. It backs up any existing storage plugins before the run and then restores after the run. Can you check and put it back?

Reply:
Lets add it back. Since even @cchang738 remembers this functionality. We can fix it later, in case its broken.